docs/api/browser-window.md

```diff
@@ -835,7 +849,7 @@ Moves window to `x` and `y`.
 
 #### `win.getPosition()`
 
-Returns an array that contains window's current position.
+Returns `Integer[]` - Contains the window's current position.
```

Comment:
I looked around and couldn't see any examples of array values being specified with any format so I went with this format as it is relatively standard as both a declaration in most languages and the typescript definition.